test/EntityFramework6.Npgsql.Tests/Support/EntityFrameworkTestBase.cs

```diff
@@ -213,6 +224,87 @@ private static DbCompiledModel CreateModel(NpgsqlConnection connection)
                 null);
             dbModel.StoreModel.AddItem(echoFunc);
 
+            var stringStoreType = dbModel.ProviderManifest.GetStoreTypes().First(x => x.ClrEquivalentType == typeof(string));
+            var modelBlogStoreType = dbModel.StoreModel.EntityTypes.First(x => x.Name == typeof(Blog).Name) as EntityType;
+            var rowType = RowType.Create(
+                modelBlogStoreType.Properties.Select(x =>
+                {
+                    var clone = EdmProperty.Create(x.Name, x.TypeUsage);
+                    clone.CollectionKind = x.CollectionKind;
+                    clone.ConcurrencyMode = x.ConcurrencyMode;
+                    clone.IsFixedLength = x.IsFixedLength;
+                    clone.IsMaxLength = x.IsMaxLength;
+                    clone.IsUnicode = x.IsUnicode;
+                    clone.MaxLength = x.MaxLength;
+                    clone.Precision = x.Precision;
+                    clone.Scale = x.Scale;
+                    clone.StoreGeneratedPattern = x.StoreGeneratedPattern;
+                    clone.SetMetadataProperties(x
+                        .MetadataProperties
+                        .Where(metadataProerty => !clone
+                            .MetadataProperties
+                            .Any(cloneMetadataProperty => cloneMetadataProperty.Name.Equals(metadataProerty.Name))));
+                    return clone;
+                }),
+                null);
+
+            var getBlogsFunc = EdmFunction.Create(
+                "StoredGetBlogsFunction",
+                "BloggingContext",
+                DataSpace.SSpace,
+                new EdmFunctionPayload
+                {
+                    ParameterTypeSemantics = ParameterTypeSemantics.AllowImplicitConversion,
+                    Schema = "dbo",
+                    IsComposable = true,
+                    IsNiladic = false,
+                    IsBuiltIn = false,
+                    IsAggregate = false,
+                    StoreFunctionName = "GetBlogsByName",
+                    ReturnParameters = new[]
+                    {
+                            FunctionParameter.Create("ReturnType1", rowType.GetCollectionType(), ParameterMode.ReturnValue)
+                    },
+                    Parameters = new[]
+                    {
+                            FunctionParameter.Create("Name", stringStoreType, ParameterMode.In)
+                    }
+                },
+                null);
+            dbModel.StoreModel.AddItem(getBlogsFunc);
+
+            var stringPrimitiveType = PrimitiveType.GetEdmPrimitiveTypes().First(x => x.ClrEquivalentType == typeof(string));
+            var modelBlogConceptualType = dbModel.ConceptualModel.EntityTypes.First(x => x.Name == typeof(Blog).Name) as EntityType;
```

Comment:
Nit: "as EntityType" is redundant and generates a useless warning. You can safely remove it.